docs/agent-api.asciidoc

```diff
@@ -729,6 +729,29 @@ Must not contain periods (`.`) as those have special meaning in Elasticsearch
 If a non-string data type is given,
 it's converted to a string before being sent to the APM Server
 
+[[apm-add-tags]]
+==== `apm.addTags()`
+
+[source,js]
+----
+apm.addTags({ [name]: value })
+----
+
+Add several tags on the current transaction.
+You can add tags multiple times.
+If an error happens during the current transaction,
+it will also get tagged with the same tags.
+
+Tags are key/value pairs that are indexed by Elasticsearch and therefore searchable (as opposed to data set via `setCustomContext()`).
+
+Arguments:
+
+* `name` - Any string.
+Must not contain periods (`.`) as those have special meaning in Elasticsearch
```

Comment:
If you indent the lines following the first line in the bullet point, I think it's easier to see the relation with the bullet points (same with the bullet point below)

Reply:
It's just copy/pasted from the `setTag` docs above. Should that change too?

Reply:
Oh, maybe there's something about asciidocs that doesn't work the way I think - or maybe I was't thinking when I wrote the above docs. Lets just leave it then. I can always go over the docs later and clean these things up